app/controllers/works_controller.rb

```diff
@@ -159,6 +159,8 @@ def drafts
       redirect_to logged_in? ? user_path(current_user) : new_user_session_path
       return
     end
+    
+    @page_subtitle = t('users.page_subtitle.drafts_page', username: @user.login)
```

Comment:
Speaking of style guidelines, please follow rubocop and use double quotes here. Generally it's a good idea to follow the linter warnings unless something looks very off about them.

Furthermore, prefer relative keys for localisation. Other spots that set the page subtitle use `t(".page_title", ...)`. Rails will deduce the path to the key automatically based on the controller and action name, so that it in the yml file it should be placed in `works.drafts.page_title`.